tests/print_events.py

```diff
@@ -1,13 +0,0 @@
-import sys
```

Comment:
why removing this ?

Reply:
I believe this script was copied from the scripts dir to the tests dir, in order to be able to invoke it in a test env. 
But I've refactored the code to handle this more gracefully and not have to duplicate the scripts for testing purposes. See https://github.com/ebarault/anyservice-awscode-docker/pull/8/files#diff-3d2084d2983188c5267b5b8203dd9941523c1ded70a99bb53fab8db6926b65eeR186

Reply:
ok